daffodil-lib/src/main/scala/org/apache/daffodil/schema/annotation/props/PropertyScoping.scala

```diff
@@ -131,6 +144,21 @@ trait FindPropertyMixin extends PropTypes {
           lr
         }
       }
+    // if this looks like it could be an expression, issue a warning
+    // but only if expressionAllowed is false
+    if(!expressionAllowed) {
+      propRes match {
+        case Found(v, _, _, _) => {
+          if(DPathUtil.isExpression(v)) {
+            SDW(WarnID.NonExpressionPropertyValueLooksLikeExpression,
+              "Property %s looks like an expression but cannot be an expression: %s", pname, v)
+          }
+        }
+        case _ => {
+          // Do Nothing
+        }
+      }
+    } 
```

Comment:
Needs a space after the ``if`` keyword above.

Also, this is just personal style preference, but all the indentions make this kind of hard to read to me. Anyone else get that, or is it just me? I'm wondering if something like this would be preferred?
```scala
if (!expressionAllowed) {
  val isExpression = propRes.toOption.map { DPathUtil.isExpression(_) }.getOrElse(false)
  schemaDefinitionWarningUnless(
    WarnID....,
    !isExpression,
   "Property ..."
  )
}